src/DynamoCore/Core/CustomNodeManager.cs

```diff
@@ -518,17 +518,40 @@ private void SetNodeInfo(CustomNodeInfo newInfo)
                 var message = string.Format(Resources.MessageCustomNodePackageFailedToLoad,
                     infoPath, newInfoPath);
 
-                // if these are different packages raise an error.
-                // TODO (for now we don't raise an error for different versions of the same package)
-                if (newInfo.PackageInfo.Name != info.PackageInfo.Name)
+               
+                //only try to compare package info if both customNodeInfos have package info.
+                if(info.IsPackageMember && info.PackageInfo != null)
```

Comment:
Doesn't one being true mean the other is true - if `IsPackageMember` is true, doesn't it imply that `PackageInfo` is non-null or is this double check just to be safe?

Reply:
yes, it's just a double check, the second check should always be true but I wanted to verify.